aarone_frontend/src/app/page_sections/home/home3/about3/about3.component.html

```diff
@@ -28,23 +28,21 @@ <h2 class="text-uppercase sep-line-one "><span class="font-weight-300 text-prima
                     <div class="m-about-containt text-black">
                         <div class="m-about-years bg-moving"
                             style="background-image:url(assets/images/background/line.png);">
-                            <span class="text-primary large-title">{{data.experience}}</span>
```

Comment:
Same for here

Reply:
Thanks for the feedback, I modified the data instead. Could you take a look, please? Thank you!